src/components/DayPickerKeyboardShortcuts.jsx

```diff
@@ -176,33 +179,36 @@ class DayPickerKeyboardShortcuts extends React.PureComponent {
 
     return (
       <div>
-        <button
-          ref={this.setShowKeyboardShortcutsButtonRef}
-          {...css(
-            styles.DayPickerKeyboardShortcuts_buttonReset,
-            styles.DayPickerKeyboardShortcuts_show,
-            bottomRight && styles.DayPickerKeyboardShortcuts_show__bottomRight,
-            topRight && styles.DayPickerKeyboardShortcuts_show__topRight,
-            topLeft && styles.DayPickerKeyboardShortcuts_show__topLeft,
-          )}
-          type="button"
-          aria-label={toggleButtonText}
-          onClick={this.onShowKeyboardShortcutsButtonClick}
-          onMouseUp={(e) => {
-            e.currentTarget.blur();
-          }}
-        >
-          <span
-            {...css(
-              styles.DayPickerKeyboardShortcuts_showSpan,
-              bottomRight && styles.DayPickerKeyboardShortcuts_showSpan__bottomRight,
-              topRight && styles.DayPickerKeyboardShortcuts_showSpan__topRight,
-              topLeft && styles.DayPickerKeyboardShortcuts_showSpan__topLeft,
-            )}
-          >
+        { renderKeyboardShortcutsButton
```

Comment:
This is a nit, but I think that the code style in react-dates is to do avoid ternaries by creating two conditions instead?

```es7
 { renderKeyboardShortcutsButton && .... }
 { renderKeyboardShortcutsButton || .... }
```

Reply:
(in jsx in general)

Reply:
also, jsx curlies should have no bounding spaces inside them